### PR TITLE
MSSQL: Adding two missing arguments for aws_db_instance: timezone & character_set_name

### PR DESCRIPTION
**Pull request by aloisbarreras**
…r_set_name

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -88,4 +88,7 @@ module "db_instance" {
   create_monitoring_role = "${var.create_monitoring_role}"
 
   tags = "${var.tags}"
```
Comment (antonbabenko) [inline main.tf]: Keep `tags` as a last line.

**Comment by aloisbarreras** [comment]
@antonbabenko good point, I didn't think about that. I took a stab at having the separate db_instance and having conditional outputs. Both mssql and mysql seem to work now. Let me know anything else that needs changed.

File: modules/db_instance/main.tf [hunk 2]
```diff
@@ -63,4 +67,57 @@ resource "aws_db_instance" "this" {
   backup_window           = "${var.backup_window}"
 
   tags = "${merge(var.tags, map("Name", format("%s", var.identifier)))}"
+
+  character_set_name = "${var.character_set_name}"
```
[the remark this answers is not in the document]
Comment (aloisbarreras) [reply modules/db_instance/main.tf]: @agomezvidalee  actually looking at the docs, character_set_name only applies to Oracle instances. I'll go ahead and remove this line.

[the remark this answers is not in the document]
Comment (aloisbarreras) [reply modules/db_instance/main.tf]: Well never mind actually. It doesn't break deploying like timezone does, so it seems safe to leave it in.

**Comment by antonbabenko** [comment]
Great work @aloisbarreras ! This PR has been released as part of [v1.15.0](https://github.com/terraform-aws-modules/terraform-aws-rds/releases/v1.15.0) release.

**Pull request merged**